src/main/java/mate/academy/spring/model/dto/request/UserRequestDto.java

```diff
@@ -0,0 +1,26 @@
+package mate.academy.spring.model.dto.request;
+
+import com.sun.istack.NotNull;
+
+public class UserRequestDto {
+    @NotNull
+    private String email;
```

Comment:
I think it is possible to add the @Email annotation here. To do this you would need to add a `Hibernate Validator` dependency.